### ISSUE TITLE
Toggle switch > `aria-readonly` not allowed in latest version of axe-core

### ISSUE THREAD

**Issue by Blackbaud-SteveBrush** [blackbaud/skyux-forms#113]
**Original design document:**
https://github.com/blackbaud/skyux-forms/issues/23

"Only use the aria-readonly attribute for elements that are not allowed to have a readonly attribute in HTML."
https://www.w3.org/TR/html-aria/#docconformance

"Only text controls can be made read-only, since for other controls (such as checkboxes and buttons) there is no useful distinction between being read-only and being disabled, so the readonly attribute does not apply."
https://html.spec.whatwg.org/multipage/input.html#attr-input-readonly

`aria-readonly` doesn't appear to be supported by the role "switch":
https://www.w3.org/TR/wai-aria-1.0/states_and_properties#aria-readonly

<img width="785" alt="Screen Shot 2020-05-06 at 8 51 43 AM" src="https://user-images.githubusercontent.com/12497062/81178860-d6bf5e00-8f76-11ea-8adf-1052ce63bfa0.png">
